src/datasource-zabbix/zabbix/zabbix.js

```diff
@@ -110,37 +110,37 @@ export class Zabbix {
     }
    ```
    */
-  testDataSource() {
-    let zabbixVersion;
-    let dbConnectorStatus;
-    return this.getVersion()
-    .then(version => {
-      zabbixVersion = version;
-      return this.login();
-    })
-    .then(() => {
-      if (this.enableDirectDBConnection) {
-        return this.dbConnector.testDataSource();
-      } else {
-        return Promise.resolve();
-      }
-    })
-    .catch(error => {
-      if (error instanceof ZabbixNotImplemented) {
-        return Promise.resolve();
-      }
-      return Promise.reject(error);
-    })
-    .then(testResult => {
-      if (testResult) {
-        dbConnectorStatus = {
-          dsType: this.dbConnector.datasourceTypeName,
-          dsName: this.dbConnector.datasourceName
-        };
-      }
-      return { zabbixVersion, dbConnectorStatus };
-    });
-  }
+  // testDataSource() {
```

Comment:
We need `testDataSource()` method implemented on the backend side.

Reply:
There is a datasource test implemented on the backend. I changed the frontend testDatasource to post to the backend here:
- https://github.com/alexanderzobnin/grafana-zabbix/pull/822/files#diff-4e2b5df346a2dd67cd90176dfb35ade5R421
- https://github.com/alexanderzobnin/grafana-zabbix/pull/822/files#diff-4e2b5df346a2dd67cd90176dfb35ade5R208-R223

and the backend is implemented here:
- https://github.com/alexanderzobnin/grafana-zabbix/pull/822/files#diff-2cb344693fe3302e52c776fc8f6e3964R41-R42
- https://github.com/alexanderzobnin/grafana-zabbix/pull/822/files#diff-5641cbffc2e0b7f7310a3c757ca99c6bR117

Unless you mean we specifically need a method called testDataSource in the backend?